packages/server/lib/browsers/protocol.js

```diff
@@ -116,8 +113,9 @@ const getWsTargetFor = (port) => {
 
     return retry()
   })
-  .tapCatch((err) => {
+  .catch((err) => {
     debug('failed to connect to CDP %o', { connectOpts, err })
+    errors.throw('CDP_COULD_NOT_CONNECT', port, err)
```

Comment:
wouldn't this change behavior in the case when `findStartPageTarget` throws an error? I mean I like this change, just needs tests update I think

Reply:
ok, will add tests. yeah, it will catch that error and wrap it, which is (i think) what we want - right now the raw error is confusing to the user, like, "what is a CRI target?"